mex/editor/components.py

```diff
@@ -83,3 +83,32 @@ def render_value(value: EditorValue) -> rx.Component:
         ),
         spacing="1",
     )
+
+
+def icon_by_stem_type(
+    stem_type: str | None,
+    **props: int | str | rx.Color,
+) -> rx.Component:
+    """Render an icon for the given stem type."""
+    # Sigh, https://reflex.dev/docs/library/data-display/icon#using-dynamic-icon-tags
+    return rx.box(
+        rx.match(
+            stem_type,
+            ("AccessPlatform", rx.icon("app_window", **props)),
+            ("Activity", rx.icon("circle_gauge", **props)),
+            ("BibliographicResource", rx.icon("book_marked", **props)),
+            ("building", rx.icon("building", **props)),
```

Comment:
Where do we use stem type "building"?

Reply:
yikes, copy&pasta 🍝 
fixed in [02ba6bd](https://github.com/robert-koch-institut/mex-editor/pull/383/commits/02ba6bd9b882baf931c77b87543e0ddf254401fc)